indexer/feature_meta.hpp

```diff
@@ -115,7 +115,6 @@ class Metadata : public MetadataBase
     FMD_EMAIL = 14,
     FMD_POSTCODE = 15,
     FMD_WIKIPEDIA = 16,
-    FMD_MAXSPEED = 17,
```

Comment:
// FMD_MAXSPEED used to be 17 but not it is stored in a section of its own.

Reply:
done